bootloader/.cargo/config

```diff
@@ -0,0 +1,7 @@
+[target.thumbv7em-none-eabi]
+linker = "arm-none-eabi-gcc"
+
+rustflags = [
+  "-C", "link-arg=-Wl,-Tlink.x",
+  "-C", "link-arg=-nostartfiles",
+]
```

Comment:
You probably want to reuse the flag (`--remap-path-prefix`) we have in Tock to ensure reproducibility in the build. Otherwise the full path is being used.
Not a breaker but good for consistency :)

Reply:
Sounds good. I introduced the flags in deploy, consistent to the other compilations.